packages/docs/page-config/ui-elements/data-table/examples/Styling.vue

```diff
@@ -1,130 +1,122 @@
 <template>
```

Comment:
This a demo I always want to scroll down. It is too complicated. Thesee values in select can be hardcoded in example, so one read "column align left - OK!". We don't need that much interactivity here.